pkg/controllers/controller_suite_test.go

```diff
@@ -6,7 +6,8 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"k8s.io/klog/v2"
+
+	klog "k8s.io/klog/v2"
```

Comment:
Why these aliases are required? Is this something from the linter?

Reply:
Yep, required by linter